src/style.css

```diff
@@ -0,0 +1,74 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.form {
+  position: relative;
+  display: flex;
+  justify-content: center;
```

Comment:
I believe display and justify-content are unnecessary here, nothing changes if you remove them